src/components/typography.js

```diff
@@ -0,0 +1,82 @@
+import styled from 'styled-components';
+import { color, typography, space, compose } from 'styled-system';
+
+const Text = styled('div')(
+  compose(
+    color,
+    typography,
+    space
+  )
+);
+Text.defaultProps = {
+  color: 'primary',
+  fontFamily: 'text',
+  fontSize: 'text',
+  textAlign: 'left',
+  padding: 0,
+  margin: 'textMargin'
+};
+
+const Heading = styled(Text)({});
```

Comment:
Do you have to include `({})`?

Reply:
I have to do something either `({})` or back-ticks. IMO, a blank object looks less strange.